onboarding.md

```diff
@@ -0,0 +1,33 @@
+# Onboarding
+
+Welcome to Hypha! This page is supposed to help those getting situated in our organization.
+
+## Contacting Hypha 
+
+We also have the following communication and contact methods:
+
+- :telephone_receiver: +1-437-887-6936 (We have specific guides for [Accessing][accessing-vm] and [Managing][managing-vm] Voicemail)
+- :mailbox: #1308-1403 Royal York Road, Etobicoke, Ontario  M9P 0A1
+- :link: [hypha.coop](https://hypha.coop)
+- :e-mail: hello@hypha.coop
+
+## Our Virtual Office
+
+In place of a physical meeting place we have virtual office comprised of the following spaces:
+
+- Matrix Chat
+- Loomio
+- GitHub Organization
+- Google Drive (with both priviledged and public folders)
+
+We also have the following "helper" tools to navigate these spaces:
+
+- [Service Inventory][service-inventory]
+- [Link Shortner][link-shortener]: [link.hypha.coop][link-shortener]
```

Comment:
Since the root just goes to a 404 or something, maybe we could instead point toward the guide for manage shortlinks, which kinda explains what they are

Reply:
```suggestion
- [Link Shortner][link-shortener]: `link.hypha.coop`
```

Thoughts on avoiding linking to the rebrandly docs here? Or does it seem helpful?